Podfile.lock

```diff
@@ -1,34 +1,40 @@
 PODS:
+  - Charts (3.4.0):
+    - Charts/Core (= 3.4.0)
+  - Charts/Core (3.4.0)
   - CorePlot (2.2)
   - EVReflection (5.10.1):
     - EVReflection/Core (= 5.10.1)
   - EVReflection/Core (5.10.1)
-  - iOSDFULibrary (4.5.0):
+  - iOSDFULibrary (4.5.1):
```

Comment:
4.6.0 is available if you want to include it

Reply:
Now I'm gonna start DFU, so thanx, it make sense.